### PR TITLE
state: fix the name of the source details

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: snapcraft/internal/sources/_bazaar.py [hunk 1]
```diff
@@ -78,8 +78,8 @@ def _get_source_details(self):
         source = self.source
 
         return {
-            'commit': commit,
-            'branch': branch,
+            'source-commit': commit,
+            'source-branch': branch,
             'source': source,
```
Comment (elopio) [inline snapcraft/internal/sources/_bazaar.py]: Joe was naming this source-source. That's wrong, right???

Comment (sergiusens) [inline snapcraft/internal/sources/_bazaar.py]: @kyrofa did the original review for that, he might know the whys and whats

Comment (elopio) [inline snapcraft/internal/sources/_bazaar.py]: From @kyrofa "yeah that's probably safe"

**Comment by elopio** [comment]
@kalikiana what I would like is that we get a suite for reproducible builds. So build once, then build again using prime/snap/snapcraft and check that the snaps are equal.

Now that this bug is fixed, I don't see a lot of value in a static check of the recorded yaml, because the unit tests are pinning the current behaviour.

**Pull request merged**